Shell/AST.cpp

```diff
@@ -367,9 +367,15 @@ RefPtr<Value> BarewordLiteral::run(RefPtr<Shell>)
 void BarewordLiteral::highlight_in_editor(Line::Editor& editor, Shell& shell, HighlightMetadata metadata)
 {
     if (metadata.is_first_in_list) {
-        editor.stylize({ m_position.start_offset, m_position.end_offset }, { Line::Style::Bold });
+        if (shell.is_runnable(m_text))
+            editor.stylize({ m_position.start_offset, m_position.end_offset }, { Line::Style::Bold });
+        else {
```

Comment:
Please either omit the braces here, or add one to the `if` above.

Reply:
Oops that wasn't intended